ui/app/registration/views/transferOutAddressFields.html

```diff
@@ -0,0 +1,69 @@
+<article class="form-field" ng-class="::{'left-form-field': $index % 2 != 0 }"
+         ng-repeat="addressLevel in ::addressLevels"
+         ng-if="addressLevel.name === 'HF_NAME' || addressLevel.name === 'HF_DISTRICT' || addressLevel.name === 'HF_PROVINCE'" addressLevel="2">
+    <div class="field-attribute" ng-if="addressLevel.name!='HF_NAME'"><label for="{{::addressLevel.addressField}}">{{::addressLevel.name | translate}}</label>
+    </div>
+    <div class="field-attribute" ng-if="addressLevel.name =='HF_NAME'"><label for="{{::addressLevel.addressField}}">{{::'REGISTRATION_TRANSFER_OUT_HF_NAME_INFO' | translate}}<span class="asterick" ng-show="isRequired" >*</span></label>
```

Comment:
This fields does not appear when the checkbox is selected.
![Screenshot from 2019-06-18 09-15-54](https://user-images.githubusercontent.com/19207552/59661193-8bf3f680-91aa-11e9-94b0-e7f4817eba7f.png)